Detectors/FIT/FV0/simulation/include/FV0Simulation/DigitizationParameters.h

```diff
@@ -0,0 +1,37 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef ALICEO2_FV0_DIGITIZATION_PARAMETERS
+#define ALICEO2_FV0_DIGITIZATION_PARAMETERS
+
+namespace o2::fv0
+{
+struct DigitizationParameters {
+  const Int_t NCellsA = 40;
```

Comment:
Could you please change ``const`` to ``static constexpr``? Also, consts (and constexpr) should not start with ``m`` but be capitalized, e.g. ``static constexpr int NPMTs = 48``

Reply:
Done